### ISSUE TITLE
Package not installed in R

### ISSUE THREAD

**Issue by kalleluunjast** [sdcTools/protoTestCensus#2]
I was trying to install this package directly in R using the instructions given in https://github.com/sdcTools/protoTestCensus/tree/master/TargetedRecordSwapping/R. The installation seemed to be successful until I encountered a problem as I was trying to display the vignette which was not found (see in installation log.txt). Also the example program does not work correctly (see in application log.txt).
What could be the problem?
[application log.txt](https://github.com/sdcTools/recordSwapping/files/2803511/application.log.txt)
[installation log.txt](https://github.com/sdcTools/recordSwapping/files/2803512/installation.log.txt)

**Comment by JohannesGuss** [comment]
I changed the installation call to 

`install.packages("devtools")
remotes::install_github(
  repo = "sdcTools/recordSwapping",
  ref="recordSwapping_prototype",
  build_opts = c("--no-resave-data", "--no-manual")
)`

Apparently there were some changes to the `devtools` package. It should work properly now.
As for the example code multiple similarity profiles are not supported for the prototype version, which is version 0.1.0, see `packageVersion("recordSwapping")`.
With the next release this feature will be added. I modified the example code to make this clear.
